CHANGELOG.md

```diff
@@ -4,6 +4,56 @@
 
 [1]: https://www.npmjs.com/package/nodejs-paginator?activeTab=versions
 
+## v0.2.0
+
+03-08-2019 12:15 PST
+
+### New Features
+- feat: handle promise based functions ([#91](https://github.com/googleapis/nodejs-paginator/pull/91))
+- refactor(ts): create generic for object streams ([#101](https://github.com/googleapis/nodejs-paginator/pull/101))
+
+### Dependencies
+- chore(deps): update dependency through2 to v3 ([#53](https://github.com/googleapis/nodejs-paginator/pull/53))
+- chore(deps): update dependency @types/is to v0.0.21 ([#55](https://github.com/googleapis/nodejs-paginator/pull/55))
+- chore(deps): update dependency gts to ^0.9.0 ([#57](https://github.com/googleapis/nodejs-paginator/pull/57))
+- fix: Pin @types/sinon to last compatible version ([#61](https://github.com/googleapis/nodejs-paginator/pull/61))
+- refactor: trim a few dependencies ([#60](https://github.com/googleapis/nodejs-paginator/pull/60))
```

Comment:
This format is pretty darn close to what you get out of `standard-version` or `semantic-release`, would be curious to talk about what specific section of the README are necessary. Currently standard-version only displays `feat`, BREAKING CHANGES and `fixes`, we'd need to add the concept of dependency updates, documentation changes, and test changes.

Reply:
Yuuuuup.  The tool that generates this today works across language.  Making the whole conventional-commits / standard-version / semantic-release toolchain work across all languages is a very noble goal for the ecosystem - and something I'd love for our team :) This is a long arc and OKR level discussion though (not for a PR)

Reply:
Releases are sort of automated (at least for Node), we use a tool to generate the release notes and publish to npm. Might be worthwhile to open an issue over at https://github.com/googleapis/releasetool